src/energiapy/graph/graph.py

```diff
@@ -25,17 +24,18 @@ def distance(u_x: float, u_y: float, v_x: float, v_y: float, method: str = 'eucl
     Returns:
         float: _description_
     """
-    if 'rectilinear':
-        distance = abs(u_x-v_x) + abs(u_y-v_y)
+    if method == 'rectilinear':
```

Comment:
wouldn't return an initialized value; if an unknown distance method is used throws a value error.